config/routes.rb

```diff
@@ -18,6 +18,7 @@
     namespace :v0 do
       resources :tokens, only: [:create]
       get 'manifests/search', to: 'manifests#search'
+      post 'manifests/validate', to: 'manifests#validate'
```

Comment:
Yes sorry for lack of clarity: by standard, I mean Rails standard. Since we are using Rails, I think it makes sense to stick with Rails standards as long as they make sense. We seem to be using Rails standard routing for this app, so I am trying to figure out the best way to stick with that standard throughout *all* of our routes.

The way I think about it in this case is:

* when we are validating a manifest, we are POSTing some data that does not update an existing object, so I think `create` is the most accurate description of this action in the context of the 7 rails default actions (GET collection/new/single object/edit, POST create, PATCH update, DELETE destroy). 
* if we agree on above, we should agree on a name of *what* is being created. It is not a manifest, certainly. It is a validation. It is a validation of a manifest, though, so I like the idea of nesting the route to make that clearer
* based on 2 bullets above, I think a logical place to put this would be `manifests/validations#create` (`create` action in a nested controller named `ValidationsController`)

What we have no works just as well, but as a precedent I think it makes sense to try and include new controller actions into one of the 7 rails defaults. There will be exceptions to this, no doubt, but I don't think this case is unusual enough to require an exception.